types/address.go

```diff
@@ -202,11 +176,6 @@ func AccAddressFromBech32(address string) (addr AccAddress, err error) {
 		return nil, err
 	}
 
-	err = VerifyAddressFormat(bz)
```

Comment:
we should still check this here, it can be private and not modifiable. This is still used by the whole ecosystem, not verifying lengths could cause unforseen issues. 

We do it in the codec now, but not anywhere else

Reply:
My proposal is to remove all this code and use the codec directly.

Reply:
but if users arent using it, then it could introduce a bug. unless we delete the whole file we cant force people to migrate

Reply:
How so? The bech32 encoding/decoding is done the same than within the codec.

Reply:
it doesnt look like we check if its len(0) or > address.Maxlen, maybe i missed it?

Reply:
We did previously here in verify address format.

It's done as well by default here.
I still don't understand why we cannot replace it immediately by the codec here, but apparently it will be done in a follow-up, so the logic will be re-added.

Reply:
I will re add the check for len(0), And pass it through the codec, which checks the maxLen